server/selinux/server/pulp-streamer.te

```diff
@@ -76,3 +80,22 @@ ifdef(`distro_fedora23', `
     fs_rw_inherited_tmpfs_files(squid_t);
     fs_read_tmpfs_files(squid_t);
 ')
+
+
+##### Pulp stuff #####
+
+## Allow access to /var/run/pulp for EL 6
+ifdef(`distro_rhel6', `
+    allow streamer_t pulp_var_run_t:file manage_file_perms;
```

Comment:
L#89 - L#91 could be turned into a Refpol statement, but I don't off hand know which one it is. We already have the same code in the `pulp-celery` SELinux policy [here](https://github.com/pulp/pulp/blob/a473ddffb18bab5ed224a40198bf4c7cfaed30cf/server/selinux/server/pulp-celery.te#L57-L60) so I say leave it as-is and we can improve both at a later time.